packages/stark-ui/src/modules/table/components/column.component.ts

```diff
@@ -130,6 +130,12 @@ export class StarkTableColumnComponent extends AbstractStarkUiComponent implemen
 	@Input()
 	public sortable = true;
 
+	/**
+	 * Display the column filter box below or above. Default: below
+	 */
+	@Input()
+	public searchFilterYPosition? = "below";
```

Comment:
This should not be a separate `input` on the component, but instead it should be put together with all the properties of the column filter here: [packages/stark-ui/src/modules/table/entities/table-column-filter.intf.ts](https://github.com/NationalBankBelgium/stark/blob/master/packages/stark-ui/src/modules/table/entities/table-column-filter.intf.ts)

Reply:
ah ok, good to know, it's a lot more clean.